src/renderer/account/components/AccountPanel/AccountPanel.js

```diff
@@ -20,7 +20,16 @@ export default class AccountPanel extends React.Component {
       <Panel className={styles.accountPanel} renderHeader={this.renderHeader}>
         <div className={styles.content}>
           NEO: {this.getBalance(NEO)}<br />
-          GAS: {this.getBalance(GAS)}
+          GAS: {this.getBalance(GAS)}<br />
+          {
+            Object.values(this.props.balances).map(({ symbol, balance }) => {
+              if (symbol === undefined || balance === '0') {
```

Comment:
I think it would make more sense to filter these out inside of `AccountPanel/index.js`.  We could use the `withProps` HOC from recompose.  This component should just be pretty dumb and render whatever it's given.